### PR TITLE
Merge docs/XBee-Implementation to main

### PR DESCRIPTION
**Pull request by Boehm-Michael**
Documentation on how to implement XBee radios into SystemReference.

### CODE REVIEW

File: docs/integration/xbee-radio-integration.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+#      rateGroup1Comp.RateGroupMemberOut[4] -> radio.run
+```
+```
+Line: 145-148 * Subject to change *
+#    connections Radio {
+#      radio.allocate -> comBufferManager.bufferGetCallee
+#      radio.deallocate -> comBufferManager.bufferSendIn
+#    }
+```
+
```
Comment (jwest115) [inline docs/integration/xbee-radio-integration.md:39]: I think we should avoid calling out specific lines, maybe we can add a comment to the section that needs to be uncommented in order to enable the driver?

File: docs/integration/xbee-radio-integration.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+#
+#    phase Fpp.ToCpp.Phases.stopTasks """
+#    comDriver.quitReadThread();
+#    """
+#
+#    phase Fpp.ToCpp.Phases.freeThreads """
+#    comDriver.join(nullptr);
+#    """
+# }
+```
```
Comment (jwest115) [inline docs/integration/xbee-radio-integration.md:80]: same comment as before

**Comment by Boehm-Michael** [comment]
@jwest115 I am more than happy to make this change!

**Comment by jwest115** [comment]
Thanks! I will merge this issue and for future work, I'll add in flags for enabling the radio stuff, that way we can avoid referencing code snippets in docs.

**Review by jwest115 — APPROVED** [review]

**Pull request merged**